tests/view_tests/tests/test_i18n.py

```diff
@@ -270,6 +270,16 @@ def test_javascript_gettext(self):
         elem = self.selenium.find_element_by_id("npgettext_plur")
         self.assertEqual(elem.text, "455 Resultate")
 
+    @modify_settings(INSTALLED_APPS={'append': ['view_tests.app1', 'view_tests.app2']})
+    @override_settings(LANGUAGE_CODE='fr')
+    def test_multiple_catalogs(self):
+        self.selenium.get('%s%s' % (self.live_server_url, '/jsi18n_multi_catalogs/'))
+
+        elem = self.selenium.find_element_by_id('app1string')
+        self.assertEqual(elem.text.encode('utf8'), b'il faut traduire cette cha\xc3\xaene de caract\xc3\xa8res de app1')
```

Comment:
I think that's the most portable way for testing this with python 2 and 3.
Another way would be to test:
```python
self.assertEqual(elem.text, u'il faut traduire cette chaîne de caractères de app1')
```
But I think, it's more portable not depending on how selenium or the current python version interpret the string.